flink-ai-flow/ai_flow/scheduler/scheduler_service.py

```diff
@@ -123,30 +147,12 @@ def resumeWorkflowScheduling(self, request, context):
                                                            error_message=traceback.format_exc()))
 
     def getWorkflow(self, request, context):
-        try:
-            rq: ScheduleWorkflowRequest = request
-            workflow_info = self._scheduler.get_workflow(rq.namespace, rq.workflow_name)
-            if workflow_info is None:
-                return WorkflowInfoResponse(
-                    result=ResultProto(status=StatusProto.ERROR,
-                                       error_message='{} do not exist!'.format(rq.workflow_name)))
-            return WorkflowInfoResponse(result=ResultProto(status=StatusProto.OK),
-                                        workflow=workflow_to_proto(workflow_info))
-        except Exception as err:
-            return WorkflowInfoResponse(result=ResultProto(status=StatusProto.ERROR,
-                                                           error_message=traceback.format_exc()))
+        return WorkflowInfoResponse(result=ResultProto(status=StatusProto.ERROR,
```

Comment:
Just to clarify, we are going to call metadata client to get, list and delete workflow, right? Will this be done in a separate PR?